redfish-core/lib/account_service.hpp

```diff
@@ -1541,6 +1541,138 @@ inline void updateUserProperties(
     });
 }
 
+inline void uploadACF(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                      const std::vector<uint8_t>& decodedAcf)
+{
+    crow::connections::systemBus->async_method_call(
+        [asyncResp](const boost::system::error_code ec,
+                    sdbusplus::message::message& m,
+                    const std::tuple<std::vector<uint8_t>, bool, std::string>&
+                        messageFDbus) {
+        if (ec)
+        {
+            BMCWEB_LOG_ERROR("DBUS response error: {}", ec);
+            if (strcmp(m.get_error()->name, "xyz.openbmc_project.Certs.Error."
+                                            "InvalidCertificate") == 0)
```

Comment:
The new CI would suggest to combine the parts of strings into one like `xyz.openbmc_project.Certs.Error.InvalidCertificate`.

I also think the better/safer comparison method seems like
```
            if ( m.get_error()->name == std::string_view("xyz.openbmc_project.Certs.Error.InvalidCertificate"))
```